Apps/phcorelight/corelight.json

```
@@ -0,0 +1,2958 @@
+                {
+                    "data_path": "action_result.data.*.name",
+                    "data_type": "string",
+                    "column_name": "File Name",
+                    "column_order": 1,
+                    "example_values": [
+                        "test_corelight.json"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.bind_dn",
+                    "data_type": "string",
+                    "example_values": [
+                        ""
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.bind_password",
+                    "data_type": "string",
+                    "example_values": [
+                        "**encrypted**AA4AAAAIAAAAAWVr4pj4c0ZHQp3YTxAl3k9s2ZCJjM2wPoQTo/wS+WmEBBhWVp4wLU5Q6ueXJNdeTMCZwWfB9KFKew54FSN6e1l2/sJpcAruLXKVSXyIRGT5upN5X3ymSb7+bbao5ooqCU0w9HfMp+hHngGJ/M/K//1G6A=="
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.enable",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.encryption",
+                    "data_type": "string",
+                    "example_values": [
+                        "starttls"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.group_base_dn",
+                    "data_type": "string",
+                    "example_values": [
+                        ""
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.group_read_only",
+                    "data_type": "string",
+                    "example_values": [
+                        ""
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.group_read_write",
+                    "data_type": "string",
+                    "example_values": [
+                        ""
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.server",
+                    "data_type": "string",
+                    "example_values": [
+                        ""
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.options.authentication.ldap.user_base_dn",
+                    "data_type": "string",
+                    "example_values": [
+                        "
```

```suggestion
                    "description": "The location for the 202",
```

At least I think that's what they were going for?

---

Fixed